### ISSUE TITLE
Element technology

### ISSUE THREAD

**Issue by jtostie** [gahansen/Albany#14]
Higher order surface elements and different integration strategies (48 point) for the composite tet.

**Comment by lxmota** [comment]
I had a conversation with John Emery. He has funds for this. @jwfoulk, @jtostie and I need to talk about this.

**Comment by mperego** [comment]
Is this something that will end up in Intrepid2?

**Comment by lxmota** [comment]
Yes, this is something that would end up being implemented in Intrepid2. I still need to talk to John Emery to see if he is OK with having this in Albany.
